reactDashboard/src/components/DataDefinitions/DataDefinitions.css

```diff
@@ -58,10 +71,28 @@
   font-weight: bold;
 }
 
+.darkmode .data-definitions .defined-term,
+.darkmode .data-definitions .definitions-heading {
+  color: #c1def1;
+}
+
+.darkmode .data-definitions hr {
+  border: 1px solid #cccccc;
```

Comment:
What hr block is this overriding? Regardless, can you just use border-color here please?

Reply:
✅ 
The default colour for that hr is too dark